### ISSUE TITLE
BLE beacon ScanRecord object has different content based on libraries version.

### ISSUE THREAD

**Issue by Maartinsh** [weliem/blessed-android#136]
Hi everyone,
Test case: the very same beacon (advertises the same message).

**1. When I test library:** `api "com.github.weliem:blessed-android:2.0.10"`

Results: You can see that ScanResults has a content length of 31.
https://i.imgur.com/EcVQjYO.png

**2. When I test library:** `api "com.github.weliem:blessed-android:2.1.6"`

Results: You can see that ScanResults has a content length of 20.
https://i.imgur.com/ahhaAx3.png

Basically, the first approach has UUID the second one does not.
